### PR TITLE
Dynamic copyright year for docs

### PR DESCRIPTION
**Pull request by bderusha**
FYI @edx/ecommerce

### CODE REVIEW

**Comment by bderusha** [comment]
FYI @edx/doc

**Comment by clintonb** [comment]
👍

File: docs/conf.py [hunk 1]
```diff
@@ -58,7 +59,7 @@
 
 # General information about the project.
 project = u'Course Discovery Service'
-copyright = u'2015, edX'
+copyright = u'{year}, edX'.format(year=datetime.date.today().year)
```
Comment (pdesjardins) [inline docs/conf.py]: In a copyright notice, the first year is important. Maybe:

copyright = u'2015-{year}, edX'.format(year=datetime.date.today().year)

Comment (bderusha) [reply docs/conf.py]: updated

**Comment by lamagnifica** [comment]
...please use edX Inc.

**Comment by bderusha** [comment]
@lamagnifica done

**Comment by lamagnifica** [comment]
:)

**Pull request merged**